src/jira/client/jira-client.ts

```diff
@@ -262,10 +297,12 @@ export const getJiraClient = async (
 					const subscription = await Subscription.getSingleInstallation(jiraHost, gitHubInstallationId, gitHubAppId);
 					await subscription?.update({ syncWarning: issueKeyLimitWarning });
 				}
+
 				const payload = {
 					builds: data.builds,
 					properties: {
-						gitHubInstallationId
+						gitHubInstallationId,
+						repositoryId
```

Comment:
Why not use tranformed repository id? I think that `transformed repo id` is necessary for GHES app?

Reply:
Okay since we use `gitHubInstallationId` as well, so GHES might be okay here, I see what you mean. 
I think there's another place do the deletion, have a look see whether we need fix that as well.
```
			installation: {
				delete: async (gitHubInstallationId: string | number) =>
```

Reply:
But the installation endpoint is to delete the entire org connection (handled by checking the passed gitHubInstallationId), this is just for a single repo that gets deleted in GH.

Reply:
Oh i mean say this is submitting with `gitHubInstallationId` and `repositoryId` (after the PR), while that installation endpoint delete only via `gitHubInstallationId`. I am not sure is an AND or OR logic, just curious whether the `installation delete endpoint` still work.

Reply:
Only the the stored property you want to delete by needs to be supplied when deleting (do not need to provider every single property an entity has when deleting. 

Have tested locally and all looks good. More info here https://developer.atlassian.com/cloud/jira/software/rest/api-group-deployments/#api-rest-deployments-0-1-bulk-post

Reply:
ha, nice 👍